doc/explanation/lxc_show_info.md

```diff
@@ -0,0 +1,49 @@
+(lxc-show-info)=
+# About `lxc` 'show' and 'info'
+For the entities managed by LXD, the `lxc` command provides a 'list' sub-command, and may provide 'show' and 'info' sub-commands.
```

Comment:
```suggestion
For the entities managed by LXD, the `lxc` command provides a 'list' sub-command, and might provide 'show' and 'info' sub-commands.
```

"may" is not wrong in this context. But in technical documentation, we're always trying to be as precise as possible and not leave anything up to interpretation - and "may" can mean a lot of different things.
A common way to distinguish is:

- "may" is for permission: You may enter the house (you're allowed to).
- "might" is for possibility: This might happen (or it might not).
- "can" is for ability: You can walk (unless you're in a wheel chair).